docs/release-notes/Percona-Everest-1.5.0-(2025-02-07).md

```diff
@@ -0,0 +1,164 @@
+    |---------|---------------------|---------|
+    | **1.**|[Role-based access control (RBAC) Generally Available (GA)](#role-based-access-control-rbac-now-generally-available-ga-in-percona-everest)|RBAC is now GA with Percona Everest 1.5.0 |
+    | **2.**|[RBAC: Integration with IdP groups](#streamlining-role-based-access-control-with-idp-group-integration)|Assign RBAC policies to user groups obtained from an external IdP|
+    | **3.**|[Operators support](https://docs.percona.com/everest/release-notes/Percona-Everest-1.4.0-%282025-01-07%29.html#__tabbed_1_4)|Support for PXC operator 1.16.1 and PSMDB operator 1.19.1|
+    | **4.**|[New features](https://docs.percona.com/everest/release-notes/Percona-Everest-1.3.0-%282024-11-18%29.html#new-features)|Check out the new features introduced in Percona Everest 1.5.0|
+    | **5.**|[Improvements](https://docs.percona.com/everest/release-notes/Percona-Everest-1.3.0-%282024-11-18%29.html#improvements)|Discover all the enhancements featured in Percona Everest 1.5.0|
+    | **6.**|[Bugs](https://docs.percona.com/everest/release-notes/Percona-Everest-1.3.0-%282024-11-18%29.html#bugs)|Find out about all the bugs fixed in Percona Everest 1.5.0|
+    | **7.**|[Known limitations](https://docs.percona.com/everest/release-notes/Percona-Everest-1.3.0-%282024-11-18%29.html#known-limitations)|Discover all the known limitations in Percona Everest 1.5.0|
+
+
+## Release highlights
+
+## Role-based access control (RBAC) now Generally Available (GA) in Percona Everest
+
+We’re delighted to unlock the GA of RBAC in Percona Everest 1.5.0!
+
+With RBAC, only authorized individuals can access specific resources or perform certain actions based on their assigned roles. This update introduces:
+
+- Granular access management: Allocate roles with detailed permissions to ensure precise access control.
+
+- Enhanced security: Restrict access to authorized users and teams only.
+
+- Better IdP integration: Integrate with your Identity Provider to streamline the authentication process and effectively manage the assignment of user roles.
+
+**Upgrade to Percona Everest today** to unlock RBAC for improved security and streamlined efficiency!
+
+For more insights into RBAC, refer to our [documentation](https://docs.percona.com/everest/administer/rbac.html).
+
+### Streamlining Role-Based Access Control with enhanced IdP group integration
+
+Starting with Percona Everest 1.5.0, you can now assign RBAC policies to user groups obtained from the external IDP. This enhancement simplifies permissions management for external users without the need for unique **sub** IDs. To use IdP groups in Percona Everest RBAC, you would need to set up the groups claim in your IdP provider configuration.
+
+During the SSO authentication, if the group’s scope is requested but the Identity Provider (IdP) lacks the necessary groups claim configuration or does not support it by default, SSO could fail or not function as expected.
+
```

Comment:
I don't think this should be part of the RNs. Maybe this should be in the documentation page and we should say something like "Configure your IdP to provide the user's groups claim by following our [documention](link)"

Reply:
Fixed